orbit/cmd/orbit/orbit.go

```diff
@@ -640,9 +640,6 @@ func (d *desktopRunner) execute() error {
 	defer close(d.executeDoneCh)
 
 	log.Info().Msg("killing any pre-existing fleet-desktop instances")
-	if err := platform.KillProcessByName(constant.DesktopAppExecName); err != nil && !errors.Is(err, platform.ErrProcessNotFound) {
-		log.Error().Err(err).Msg("killProcess")
-	}
```

Comment:
Did you mean to replace by `platform.GracefulProcessKillByName`? (As the above log says, we are doing this to kill any lingering fleet desktop instances from previous runs, caused by e.g. bugs)

Reply:
I update the description of the PR to comment on why this function was initially removed